api/api/routers/runs_v1.py

```diff
@@ -181,6 +181,11 @@ def from_domain(cls, error: ErrorResponse.Error):
 
     conversation_id: str | None
 
+    metadata: dict[str, Any] | None = Field(
```

Comment:
@pierrevalade only API change. 
Added metadata field to RunV1 object which is returned by `/{agent_id}/runs/latest` and the run by ID endpoint